### PR TITLE
feature: align cover svg to viewer

### PR DESCRIPTION
**Pull request by wolasss**
These changes allow to call:

`viewer.fitToViewer(ALIGN_COVER, ALIGN_COVER)` so that the svg will always take the whole viewport and match the viewport by the short edge. 

Please review and let me know if there is something else I must add

### CODE REVIEW

**Comment by wolasss** [comment]
It fixes https://github.com/chrvadala/react-svg-pan-zoom/issues/29

**Comment by wolasss** [comment]
@chrvadala hey, any chances this will be merged soon? :)

File: docs/documentation.md [hunk 1]
```diff
@@ -107,8 +107,8 @@ import {
   //components
   ReactSVGPanZoom,             //Controlled component
   UncontrolledReactSVGPanZoom, //Uncontrolled component
-  
-  //functions to manipulate the state. The signature is quite similar to component methods, expect for the first parameter that is the current value. These are usefull only for some advanced integrations (e.g. reducers). 
+
+  //functions to manipulate the state. The signature is quite similar to component methods, expect for the first parameter that is the current value. These are usefull only for some advanced integrations (e.g. reducers).
```
Comment (chrvadala) [inline docs/documentation.md]: This PR changes a lot of blank line. Can you clean this file and submit only the changed text?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,19103 @@
+{
```
Comment (chrvadala) [inline package-lock.json]: Can you remove this file from PR? It might overlap with `yarn.lock`

**Comment by chrvadala** [comment]
> Hi, thanks for your contributions.
> I added few feedbacks inline; moreover, can you provide some unit tests?
> 
> Finally, let me to suggest you to align your PR with the add-test branch https://github.com/chrvadala/react-svg-pan-zoom/tree/add-tests, because I'll merge it in the next few days

quick update: add-tests is merged! you can sync your branch with master

**Comment by wolasss** [comment]
Ok, thanks, I'll address all the issues after the weekend

**Comment by wolasss** [comment]
@chrvadala I added some unit tests - for two scenarios where rect > viewport  and the opposite. Both cases seem to be working fine. I also removed the package-lock.json and removed the blank lines.

**Comment by chrvadala** [comment]
Released with version 3.8.0
Thanks for your contribution, I added you in contributors list

**Pull request merged**